src/Microsoft.DotNet.Build.Tasks/PackageFiles/PerfTesting.targets

```diff
@@ -1,32 +1,31 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
-    <!-- Perf Runner NuGet package paths -->
+  <!-- Perf Runner NuGet package paths -->
+  <PropertyGroup>
+   <XunitPerfAnalysisPackageVersion Condition="'$(XunitPerfAnalysisPackageVersion)' == ''">1.0.0-alpha-build0037</XunitPerfAnalysisPackageVersion>
```

Comment:
I do think it is worth throwing an error when trying to run performance tests if the version isn't found.